software/praktikumsplaner.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Praktikumsplaner
+developer: LHM
+code: https://github.com/it-at-m/Praktikumsplaner
+codelinktext: github.com/it-at-m/praktikumsplaner
+licensingmodel: open source
+logo: /inhouse/praktikumsplaner_logo.png
+license: MIT
+tags:
+- application
+- eigenentwicklung
+- start2023
+sortingPriority: 3
+---
+The __Praktikumsplaner__ offers a way to organize junior staff and their internships during their vocational training.
+
+---
+## About the project
+
+The Praktikumsplaner, a project launched in collaboration between the Competence Center of Software Engineering (CCSE) and the InnovationLab, aims to support the training management in the allocation of junior staff.
+It offers trainers the opportunity to report their available internships to the training management.
+They can then use the application to assign junior staff to the reported internships using drag-and-drop functionality.
+The internship planner helps to maintain a good overview in order to assign junior staff to the most suitable positions.
+
+![Screenshot of the assignment page of the Praktikumsplaners with sample data](/public/inhouse/Screenshot_Praktikumsplaner.png)
```

Comment:
`/public/` is not used hier (see [zeitmanagementsystem.md](https://github.com/it-at-m/opensource.muenchen.de/blob/main/software/zeitmanagementsystem.md?plain=1#L35))